templates/archive.php

```diff
@@ -1,18 +1,16 @@
-<header class="title col-md-9">
-    <hgroup>
-        <h1><?php echo h()->w_template_title(); ?></h1>
-        <?php if (category_description()) :?>
-            <h2><?php echo category_description(); ?></h2>
-        <?php endif; ?>
-    </hgroup>
+<header>
+    <h1><?php echo w_template_title(); ?></h1>
+    <?php if(category_description()) :?>
+      <h2><?php echo category_description(); ?></h2>
+    <?php endif; ?>
 </header>
 
 <?php if (have_posts()) : ?>
-    <?php while (have_posts()) : the_post(); ?>
+  <?php while (have_posts()) : the_post(); ?>
```

Comment:
I've gone through everything in `templates/` and fixed the indentation. So you may need to update the pagination in a few files here.